JavaScript/Merging_PDF/README.md

```diff
@@ -0,0 +1,6 @@
+# Merging PDF
+
+This is the javascript code to merge two pdf from local repository, 
+this code just uses pdflib package to load a file and make a change like given below
+
+![image](output.png)
```

Comment:
Please remove the output.png, rather upload it on some image hosting site, paste the link here.